### ISSUE TITLE
msg.parseEmotes() does not work in the onAction event handler.

### ISSUE THREAD

**Issue by chevtek** [twurple/twurple#409]
### Code

```ts
chatClient.onAction((channel, user, text, msg) => {
  console.log(msg.parseEmotes())
})
```

### Expected behavior

In the regular `onMessage` handler `msg.parseEmotes` works as expected and produces the following result given the chat input `"test TwitchLit TwitchLit test"`. `TwitchLit` being a twitch emote. I would expect this same output when using `"/me test TwitchLit TwitchLit test"`.

```json
[
  { "type": "text", "position": 0, "length": 5, "text": "test " },
  {
    "type": "emote",
    "position": 5,
    "length": 9,
    "id": "166263",
    "name": "TwitchLit",
    "displayInfo": ["ChatEmote#166263 - please check https://twurple.js.org/reference/common/classes/ChatEmote.html for available properties"]
  },
  { "type": "text", "position": 14, "length": 1, "text": " " },
  {
    "type": "emote",
    "position": 15,
    "length": 9,
    "id": "166263",
    "name": "TwitchLit",
    "displayInfo": ["ChatEmote#166263 - please check https://twurple.js.org/reference/common/classes/ChatEmote.html for available properties"]
  },
  { "type": "text", "position": 24, "length": 5, "text": " test" }
]
```

### Actual Behavior

In the `onAction` handler the `text` string parameter on the handler is totally fine and contains `"test TwitchLit TwitchLit test"` as expected. However `msg.content.value` contains extra metadata: `"\x01ACTION test TwitchLit TwitchLit test\x01"`. Since `msg.parseEmotes` utilizes `msg.content.value` it completely screws up the parsed output:

```json
[
  { "type": "text", "position": 0, "length": 5, "text": "\x01ACTI" },
  {
    "type": "emote",
    "position": 5,
    "length": 9,
    "id": "166263",
    "name": "ON test T",
    "displayInfo": ["ChatEmote#166263 - please check https://twurple.js.org/reference/common/classes/ChatEmote.html for available properties"]
  },
  { "type": "text", "position": 14, "length": 1, "text": "w" },
  {
    "type": "emote",
    "position": 15,
    "length": 9,
    "id": "166263",
    "name": "itchLit T",
    "displayInfo": ["ChatEmote#166263 - please check https://twurple.js.org/reference/common/classes/ChatEmote.html for available properties"]
  },
  { "type": "text", "position": 24, "length": 14, "text": "witchLit test\x01" }
]
```

### Workaround

If anyone else is having this same issue before it's resolved, there is a simple fix. Simply override the `msg.content.value` with the properly parsed `text` parameter. Then `msg.parseEmotes` works as expected.

```ts
twitchClient.onAction((channel, user, text, msg) => {
  msg.content.value = text // <--- workaround
  const parsedText = msg.parseEmotes()
})
```

### Environment

- Version: @twurple/chat v5.2.5
- Node version: 18.12.0 (current LTS)
- Browser version: N/A, only node
- Operating system: Windows Subsystems for Linux (and node:18.12.0-alpine in Docker)

**Comment by d-fischer** [comment]
Fixed in 5.2.6.

**Comment by chevtek** [comment]
Quick turnaround! Thanks 😊
